interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is usually before an argument.
```

Comment:
Make sure to expand on your answers providing definitions and examples if necessary.  This is supposed to mimic how you would respond in an interview, and one line answers may come off uninterested.